src/exceptions/LLVM-exception.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="LLVM-exception" name="LLVM Exceptions to the Apache 2.0">
+      <crossRefs>
+         <crossRef>http://lists.llvm.org/pipermail/llvm-dev/2017-August/116266.html</crossRef>
+      </crossRefs>
+      <notes>This exception was created specifically to be used with Apache-2.0</notes>
+	  <titleText>
+		<p>
+		<optional>---- </optional>LLVM Exceptions to the Apache 2.0 License<optional> ----</optional>
```

Comment:
I'd rather have the dashes inside an empty `<alt>` (see previous discussion [here][1] and [here][2]), because I see the dashes as header markup.  I'd much rather they *not* be part of our canonical HTML, and instead would rather use `<h1>` or other canonical HTML markup for “this is a header”.  We don't have “this is a header” markup in our XML schema yet, but I'd still rather not have this be a part of the canonical HTML we display under spdx.org/licenses/.

[1]: https://github.com/spdx/license-list-XML/pull/568#discussion_r158869521
[2]: https://github.com/spdx/license-list-XML/pull/568#issuecomment-354181552

Reply:
leaving as optional - we can update to <alt> later if need be, but this is not substantive and the dashes are taken from the email confirming the new exception, which is not yet showing up in the source repo. We'll update the URL when that hits and can review this then.

Reply:
> leaving as optional - we can update to later if need be, but this is not substantive…

That's exactly why I prefer `<alt>`.  Why should we have an opinion on the number of dashes?